<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RiskCalcMainWindow</class>
 <widget class="QMainWindow" name="RiskCalcMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Risk Calculator (Qt)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_root">
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="leftMargin">
     <number>14</number>
    </property>
    <property name="topMargin">
     <number>14</number>
    </property>
    <property name="rightMargin">
     <number>14</number>
    </property>
    <property name="bottomMargin">
     <number>14</number>
    </property>
    <item>
     <widget class="QFrame" name="top_frame">
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_top">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="lbl_threshold">
         <property name="text">
          <string>TrÃ¶skelset:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="threshold_combo">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacer_top_1">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="lbl_budget">
         <property name="text">
          <string>Budget:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="budget_edit">
         <property name="maximumSize">
          <size>
           <width>140</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>1000000</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_currency">
         <property name="text">
          <string>Valuta:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="budget_currency">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacer_top_expand">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabs">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_questionnaire">
       <attribute name="title">
        <string>FrÃ¥geformulÃ¤r</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_questionnaire">
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QFrame" name="form_frame">
          <property name="frameShape">
           <enum>QFrame::Shape::StyledPanel</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_form">
           <property name="spacing">
            <number>10</number>
           </property>
           <property name="leftMargin">
            <number>10</number>
           </property>
           <property name="topMargin">
            <number>8</number>
           </property>
           <property name="rightMargin">
            <number>10</number>
           </property>
           <property name="bottomMargin">
            <number>8</number>
           </property>
           <item>
            <widget class="QLabel" name="lbl_form">
             <property name="text">
              <string>FormulÃ¤r:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="form_combo">
             <property name="minimumSize">
              <size>
               <width>260</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="spacer_form_expand">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Policy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>10</width>
               <height>10</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="scroll">
          <property name="frameShape">
           <enum>QFrame::Shape::NoFrame</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="questions_host">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>1066</width>
             <height>421</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="questions_layout">
            <property name="spacing">
             <number>10</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_manual">
       <attribute name="title">
        <string>Manuell</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_manual">
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QGroupBox" name="manual_box">
          <property name="title">
           <string>Manuell inmatning (MonteCarloRange: min / probable / max)</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_manual" columnstretch="0,0,0,0">
           <property name="horizontalSpacing">
            <number>10</number>
           </property>
           <property name="verticalSpacing">
            <number>6</number>
           </property>
           <item row="3" column="1">
            <widget class="QLineEdit" name="lm_min">
             <property name="placeholderText">
              <string>min</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="lbl_vuln">
             <property name="text">
              <string>Vulnerability (%)</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="vuln_min">
             <property name="placeholderText">
              <string>min</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QLineEdit" name="vuln_prob">
             <property name="placeholderText">
              <string>probable</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lef_min">
             <property name="placeholderText">
              <string>min</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLineEdit" name="lef_prob">
             <property name="placeholderText">
              <string>probable</string>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QLineEdit" name="lm_max">
             <property name="placeholderText">
              <string>max</string>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QLineEdit" name="vuln_max">
             <property name="placeholderText">
              <string>max</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="lbl_lm">
             <property name="text">
              <string>Loss Magnitude (%budget/event)</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QLineEdit" name="lm_prob">
             <property name="placeholderText">
              <string>probable</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QLineEdit" name="lef_max">
             <property name="placeholderText">
              <string>max</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="lbl_prob_hdr">
             <property name="text">
              <string>Probable</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="lbl_min_hdr">
             <property name="text">
              <string>Minimum</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLabel" name="lbl_max_hdr">
             <property name="text">
              <string>Maximum</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="lbl_lef">
             <property name="text">
              <string>Loss Event Frequency (events/year)</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="spacer_manual_bottom">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Policy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="result_panel">
      <property name="title">
       <string>Resultat</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_results">
       <property name="spacing">
        <number>8</number>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_actions">
         <item>
          <widget class="QPushButton" name="calc_btn">
           <property name="text">
            <string>ðŸ§® BerÃ¤kna risk</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacer_actions">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Policy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>10</width>
             <height>10</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_result_cols">
         <property name="spacing">
          <number>18</number>
         </property>
         <item>
          <widget class="QGroupBox" name="box_scale">
           <property name="title">
            <string>Kvalitativt</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_scale" columnstretch="0,0">
            <property name="horizontalSpacing">
             <number>12</number>
            </property>
            <property name="verticalSpacing">
             <number>6</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="lbl_title_sann">
              <property name="text">
               <string>Sannolikhet:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="lbl_sannolikhet">
              <property name="text">
               <string>â€”</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="lbl_title_kons">
              <property name="text">
               <string>Konsekvens:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="lbl_konsekvens">
              <property name="text">
               <string>â€”</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="lbl_title_risk">
              <property name="text">
               <string>Risk:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="lbl_risk">
              <property name="text">
               <string>â€”</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="box_quant">
           <property name="title">
            <string>Kvantitativt</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_quant" columnstretch="0,0">
            <property name="horizontalSpacing">
             <number>12</number>
            </property>
            <property name="verticalSpacing">
             <number>6</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="lbl_title_lef">
              <property name="text">
               <string>LEF:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="lbl_lef_val">
              <property name="text">
               <string>â€”</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="lbl_title_lm">
              <property name="text">
               <string>Loss Magnitude:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="lbl_loss_magnitude">
              <property name="text">
               <string>â€”</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="lbl_title_ale">
              <property name="text">
               <string>ALE:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="lbl_ale">
              <property name="text">
               <string>â€”</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
